cli/cli.hs

```diff
@@ -203,7 +203,7 @@ main = do
     } }
   case cmd of
     Version {} ->putStrLn getFullVersion
-    Symbolic {} -> do
+    Symbolic' {} -> do
```

Comment:
This breaks the cli interface and turns `hevm symbolic` into `hevm symbolic'`. We should keep this as is

Reply:
fixed